### PR TITLE
Pushing all changes to my branch

### PR DESCRIPTION
**Pull request by ValiaLohaza**

### CODE REVIEW

File: src/test/java/stepdefinitions/VerifySubmodulesSteps.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+import utils.ConfigReader;
+import utils.DriverHelper;
+
+public class VerifySubmodulesSteps {
+
+    WebDriver driver = DriverHelper.getDriver();
+    LoginPage loginPage = new LoginPage(driver);
+    SubmodulesPage submodulesPage;
+
+    @Given("employee navigates to right webpage")
```
Comment (diamond2602) [inline src/test/java/stepdefinitions/VerifySubmodulesSteps.java]: Can you please explain please what this method does?

Comment (ValiaLohaza) [reply src/test/java/stepdefinitions/VerifySubmodulesSteps.java]: Do you mean @Given? Navigates to url or you meant submodulesPage?

Comment (diamond2602) [reply src/test/java/stepdefinitions/VerifySubmodulesSteps.java]: I was referring to-->  ConfigReader.readProperty("techtorial_CRM_URL");

**Review by diamond2602 — APPROVED** [review]
LGTM

**Pull request merged**